standalone/runtime/tests/test_redeem.rs

```diff
@@ -2027,6 +2027,14 @@ fn get_additional_collateral(vault_id: &VaultId) {
     ));
 }
 
+fn get_a_bit_of_additional_collateral(vault_id: &VaultId) {
```

Comment:
Minor nit: I'd move this small helper directly into the test. That'll help a bit with understanding how much is added as the function isn't reused as far as I can see.